IronFoundry.Warden/Tasks/ProcessCommand.cs

```diff
@@ -66,6 +94,89 @@ protected TaskCommandResult RunProcess(string workingDirectory, string executabl
             return new TaskCommandResult(exitCode, null, null);
         }
 
+        /// <summary>
+        /// Create a log in the container holding the id of the process.
+        /// </summary>
+        /// <returns></returns>
+        private TempFile CreatePidLog(int pid)
+        {
+            var tempFile = container.FileInContainer(PidLogRelativePath);
+
+            System.IO.File.WriteAllText(tempFile.FullName, pid.ToString(), Encoding.ASCII);
+            
+            return tempFile;
+        }
+
+        /// <summary>
+        /// Create a log in the container with the environment variables used to start the process.
+        /// </summary>
+        private TempFile CreateProcessEnvLog(IDictionary<string, string> processEnv)
+        {
+            var tempFile = container.FileInContainer(EnvLogRelativePath);
+            
+            var envLogBuilder = new StringBuilder();
+            foreach (var kv in processEnv)
+            {
+                string line = string.Format("{0} = {1}", kv.Key, kv.Value);
```

Comment:
Is this the same format that the Powershell script was writing?  Specifically, with the spaces around the `=` sign?